dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -669,13 +684,14 @@ const SignInGateSelectorAuxia = ({
 	return (
 		<>
 			{!isGateDismissed &&
-				auxiaGateDisplayData?.auxiaData.userTreatment !== undefined && (
+				auxiaGateDisplayData?.auxiaData.userTreatment !== undefined &&
+				abTestModified && (
 					<ShowSignInGateAuxia
 						host={host}
 						signInUrl={signInUrl}
 						// eslint-disable-next-line @typescript-eslint/strict-boolean-expressions -- Odd react types, should review
 						setShowGate={(show) => setIsGateDismissed(!show)}
-						abTest={abTest}
+						abTest={abTestModified}
```

Comment:
not important but you could avoid extra `useState` if you just make it a function, e.g.
`const buildAbTest = (userTreatment: AuxiaAPIResponseDataUserTreatment): AbTest => ...`
and
`abTest={buildAbTest(data.auxiaData.userTreatment.treatmentId)}`

Reply:
I love it. Let me do that...